web/admin/src/pages/document/component/ImportDoc/URL.tsx

```diff
@@ -64,7 +64,7 @@ const URLImport = ({
     const urls = url.split('\n');
     for (const url of urls) {
       newQueue.push(async () => {
-        const { title, content } = await postApiV1CrawlerScrape({ url, kb_id });
+        const { title, content } = await postApiV1CrawlerScrape({ url });
```

Comment:
在URL.tsx文件中，API调用参数可以进一步优化以提高代码可读性。

```suggestion
// 建议明确参数名称，提高可读性
const { title, content } = await postApiV1CrawlerScrape({ 
  url,
  // kb_id参数用于指定知识库ID
  // kb_id 
});